mammoth/opts.py

```diff
@@ -584,6 +538,12 @@ def _add_train_general_opts(parser):
         choices=['none', 'all', 'states', 'keep_states'],
         help="Optimization resetter when train_from.",
     )
+    group.add(
```

Comment:
not sure this is the best thing to add in an official lib, but hey, whatever. Happy to discuss it if you want a bone to be picked